src/Collections-Abstract/Collection.class.st

```diff
@@ -228,11 +242,13 @@ Collection >> anySatisfy: aBlock [
 
 { #category : #converting }
 Collection >> asArray [
-	"Answer an Array whose elements are the elements of the receiver.
+	"Answer an new Array whose elements are the elements of the receiver.
```

Comment:
I'm not sure about the new part because if you send #(1 2 3) asArray it probably returns self so just the receiver and not a copy.

Reply:
You are right. The method builds a new array, but is redefined in Array to return self.
So I will assume for now that all the `asX` methods try to return self if applicable.